index.js

```diff
@@ -1,11 +1,9 @@
 var path = require('path')
-var fs = require('fs')
+var fs = require('fs-extra')
 var os = require('os')
 
 var download = require('electron-download')
 var extract = require('extract-zip')
-var mkdirp = require('mkdirp')
```

Comment:
Using fs-extra was approved on first iteration (no negative comments), so, I have decided to cleanup — no need to to use mkdirp/rimraf explicitly if we use fs-exra